src/styles/blocks/catalog.scss

```
@@ -17,9 +18,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: transform $color-title-smoothly;
+
+    &:hover {
+      color: #34568b;
+      transform: scale(120%);
+    }
```

You change color on hover too, so transition should have color property as well.
```suggestion
    transition: transform $color-title-smoothly, color $color-title-smoothly;

    &:hover {
      color: #34568b;
      transform: scale(120%);
    }
```